package.json

```diff
@@ -22,7 +23,10 @@
   },
   "homepage": "https://github.com/highlightjs/highlightjs-vue#readme",
   "dependencies": {
-    "highlight.js": "^9.15.10",
+    "@types/highlight.js": "^9.12.3",
```

Comment:
I don't know typescript, but wondering if this should also be a dev dependency?

Reply:
Not sure about this, but I believe it should be `"dependencies"`, as users don't have to install `@types/highlight.js` themselves, when needing types.

Reply:
@types its only for TypeScript types and must be dev dependency. 
highlight.js is necessary be installed also as dependency.

Reply:
Thanks. It seems so. This is relevant and up to date -- https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies